includes/gesso.drush.inc

```diff
@@ -38,7 +41,6 @@ function gesso_drush_help($section) {
  * Implements drush_hook_COMMAND_validate().
  */
 function drush_gesso_validate($name = NULL) {
-  $description = drush_get_option('description');
```

Comment:
Don’t we still need this line since we still have a description option?

Reply:
The variable `$description` isn't used in this function, though

Reply:
Ah, then it’s definitely okay to remove that line.